oaknational/Oak-Web-Application
# Feat/restyle search bar LESQ-463

## Re-style search bar component

Music year: 1956

- Changes styling of input box and button
- moves filter and changes color also 
- add mini-menu svg to cma
- removes title and text

## Acceptance Criteria:

- [x]  Remove instructional copy from the top of the page that says “*Search for topics and key words to explore thousands of lessons with adaptable teaching resources*”
- [x]  Remove the inputted search term (or the word “search” if no term is entered) at the top of the page above the search box (search box should always be the first thing on the page)
- [x]  remove yellow background behind search box on search page
- [x]  New styling of icon (rectangular background)

## Issue(s)

Fixes #LESQ-463

## How to test

1. Go to https://deploy-preview-2113--oak-web-application.netlify.thenational.academy
2. Search bar on home and search page should be input component
3. Search should continute to work as normal

## Screenshots

How it used to look:
<img width="569" alt="Screenshot 2023-11-27 at 12 09 31" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/15305452-3ef8-4978-b2e7-9f7ac40a4f77">:
<img width="1115" alt="Screenshot 2023-11-27 at 12 09 37" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/a88cd8d6-b726-46a1-9f66-142348e584c1">


How it should now look:
<img width="709" alt="Screenshot 2023-11-27 at 12 09 07" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/ed6d743f-e7fd-4fdf-b684-73077c5869ab">
<img width="978" alt="Screenshot 2023-11-27 at 12 09 19" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/9783cd89-0134-4c28-b218-a7a1cfbbc8e1">


## Checklist
